src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -51,12 +51,16 @@ function getExtraOptions(extraOptions: string) {
 function RevisionHeader(props: RevisionHeaderProps) {
   const { header } = props;
   const extraOptions = getExtraOptions(header.extra_options);
+  const shortHash = truncateHash(header.new_rev);
   return (
-    <TableRow className='revision-header'>
+    <TableRow
+      className='revision-header'
+      data-testid={`revision-header-${shortHash}`}
```

Comment:
optional: using test-ids is not recommended by testing-library developers, instead it's better to directly look for actual displayed text using `getByText` or `getByRole` (or their `findBy*`counterparts).

For example in this case I believe that in the test you could do a `screen.getByText(shortHash)` instead.

Reply:
Just so that you see this, this comment is still there. It's still optional to fix and could be done in a follow-up.